sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,672 @@
+  extensions that provide different ways of allocating and accessing device
+  local memory (i.e. shared by all work-items within a work-group).
+  intel/llvm#15061, intel/llvm#16325
+  - The former is only supported on CUDA backend
+- Introduced and implemented
+  [`sycl_ext_intel_kernel_queries`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/supported/sycl_ext_intel_kernel_queries.asciidoc)
+  extension. intel/llvm#16834
+- Implemented proposed
+  [`sycl_ext_intel_event_mode`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/proposed/sycl_ext_intel_event_mode.asciidoc)
+  extension. intel/llvm#16108
+- Completed implementation of
+  [`sycl_ext_oneapi_launch_queries`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/proposed/sycl_ext_oneapi_launch_queries.asciidoc)
+  extension. intel/llvm#16709, intel/llvm#16051
+- Completed implementation of the
+  [`sycl_ext_oneapi_kernel_arg_properties`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_arg_properties.asciidoc)
+  extension by implementing missing `unaliased` property. intel/llvm#16090
+  - It used to be called `restrict` in previous versions of the extension, but
+    a renaming was done to avoid conflict with C99 `restrict` type qualifier.
+    intel/llvm#16814
+- Introduced and implemented the
+  [`sycl_ext_oneapi_num_compute_units`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/supported/sycl_ext_oneapi_num_compute_units.asciidoc)
+  extension. intel/llvm#16293, intel/llvm#16538
+
+### New compiler options
+
+- Added support for ``-f[no]-offload-fp32-prec-div` and
+  `-f[no-]-offload-fp32-prec-sqrt` compiler flags to control precision of
+  floating-point division and square root. intel/llvm#15836, intel/llvm#16107,
+  intel/llvm#16993, intel/llvm#17044, intel/llvm#17033, intel/llvm#16942,
+  intel/llvm#16714, intel/llvm#17393, intel/llvm#17253
+
+### Sanitizers
+
+#### Memory Sanitizer
+
+- Introduced memory sanitizer support. intel/llvm#15955, intel/llvm#16427,
+  intel/llvm#16478, intel/llvm#16935, intel/llvm#16535, intel/llvm#16477,
+  intel/llvm#16567, intel/llvm#16526, intel/llvm#16678, intel/llvm#16566,
+  intel/llvm#16619, intel/llvm#16705
+
+  It features:
+  - Checking for uses of uninitalized values in private memory. intel/llvm#17309
+  - Checking for uses of unitialized values in local memory, such
+    as `local_accessor` or `group_local_memory`. intel/llvm#17180,
+    intel/llvm#17054
+  - Sanitizing USM operations like `memset` or `memcpy`. intel/llvm#16511
+
+#### Thread Sanitizer
+
+- Introduced thread sanitizer support for device code. intel/llvm#17345,
```

Comment:
change line 138 to this following: 
"Introduced Thread sanitizer support for SYCL and OpenMP C/C++ device code to detect data races. The USM and device global memory are checked in this release."